react-multiple-components/my-app/src/Indicators.tsx

```diff
@@ -0,0 +1,13 @@
+type Prop = {
+  count: number;
+  index: number;
+};
+
+export function Indicators({ count }: Prop) {
+  const rowOfButtons: JSX.Element[] = [];
+  for (let i = 0; i < count; i++) {
+    rowOfButtons.push(<button key={i}>{i}</button>);
```

Comment:
Now you need to change the background color of the button that matches `index` to be light blue. You can do that by comparing `i` with `index`.